Patches/Xenn/ThingDefs_Misc/Xenn_Weps.xml

```diff
@@ -54,33 +54,30 @@
 							<capacities>
 								<li>Blunt</li>

```

Comment:
It's no longer necessary for ranged weapons to have individualized melee attack tools calculated. As this appears to be configured like a standard, non-pistol, non-heavy weapon firearm, you can just copy-paste the melee attack stats CE gives the various shotguns and rifles in `.../Patches/Core/ThingDefs_Misc/Weapons_Guns`.

Reply:
Fixed in 615aa79